WordPress/Classes/ViewRelated/Reader/Subscriptions/ReaderSubscriptionHelper.swift

```diff
@@ -32,7 +28,8 @@ struct ReaderSubscriptionHelper {
         try await withUnsafeThrowingContinuation { continuation in
             let service = ReaderSiteService(coreDataStack: contextManager)
             service.followSite(by: url, success: {
-                postSiteFollowedNotification(siteURL: url)
+                ReaderTopicService(coreDataStack: contextManager)
```

Comment:
I lost `.ReaderSiteFollowed` notification handler at some point. It's not needed because the list is data-driven. It just needs to fetch the new "topic".

The existing `service.topic(withSiteURL: siteURL)` never worked because the site it returns has `following` set to `false`, so it never appears in the list. The safest option is to just reload the list.